### PR TITLE
Livequery

### PR DESCRIPTION
**Pull request by luigidellaquila**
Hi @phpnode 

this is a pull request for live query support.
Please review this (especially connection.js line 360...) and if you need any info just let me know

Thanks a lot

### CODE REVIEW

File: lib/transport/binary/protocol28/operation.js [hunk 1]
```diff
@@ -394,10 +396,49 @@ Operation.prototype.consume = function (buffer, offset) {
...
+        ).toNumber();
+        offset+=8;
+        var contentLenght = buffer.readInt32BE(offset);
+        offset+=4;
+
+        var asString = buffer.toString('utf8', offset, offset + contentLenght);
+        offset += contentLenght;
+        var content = deserializer.deserialize(asString, this.data.transformerFunctions);
+
+        var obj = {};
```
Comment (phpnode) [inline lib/transport/binary/protocol28/operation.js]: `obj` is defined twice in this function so the tests are failing, I would remove the `obj` declaration and replace this with:
```js
return [
  Operation.LIVE_RESULT,
  token,
  operation,
  {
    content: content,
    type: 'd',
    cluster: clusterId,
    position: clusterPosition,
    version: version
  },
  offset
];
```

Comment (luigidellaquila) [inline lib/transport/binary/protocol28/operation.js]: Hi Charles,

thank you very much, I'll fix it and will send another PR soon

Thanks

Luigi


2015-04-27 20:42 GMT+02:00 Charles Pick <notifications@github.com>:

**Comment by luigidellaquila** [comment]
Hi @phpnode 

I just applied the suggested fix. Let me know if the rest is ok

Thanks

Luigi

File: lib/transport/binary/protocol28/operation.js (hunk 1 above)
Comment (luigidellaquila) [inline lib/transport/binary/protocol28/operation.js]: Hi Charles,

Fix applied, let me know if the rest is ok

Thank you very much!

Luigi



2015-04-28 11:29 GMT+02:00 Luigi Dell'Aquila <luigi.dellaquila@gmail.com>:

**Comment by phpnode** [comment]
@luigidellaquila thanks!

**Comment by gauravdhiman** [comment]
Can you kindly update the Readme with an example to use merged feature of LiveQuery.
I would like to use the LiveQuery feature for building Meteor package to make Meteor and OrientDB work together.

**Comment by phpnode** [comment]
@luigidellaquila please could you open another PR with that? :)

**Comment by luigidellaquila** [comment]
Of course I will ;-)

Thanks

Luigi


2015-05-20 15:57 GMT+02:00 Charles Pick <notifications@github.com>:

**Pull request merged**